llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Utils/Utils.cpp

```diff
@@ -80,6 +80,28 @@ Value createValueFrom2DConstant(const float *val, int64_t rows, int64_t cols,
                RankedTensorType::get(shape, rewriter.getF32Type()), vector));
 }
 
+IntegerAttr toIntegerAttr(MLIRContext *context, int64_t i) {
+  return IntegerAttr::get(IntegerType::get(context, 64), APInt(64, i));
+}
+
+SmallVector<OpFoldResult> toOpFoldResult(MLIRContext *context,
+                                         ArrayRef<int64_t> array) {
+  SmallVector<OpFoldResult> result;
+  for (int64_t i : array) {
+    result.emplace_back(toIntegerAttr(context, i));
+  }
+  return result;
+}
+
+int64_t getAttrValueOrDynamic(OpFoldResult ofr) {
```

Comment:
You could use this https://github.com/llvm/llvm-project/blob/6a70b846a44189481656f0cb5201add396a1ed41/mlir/include/mlir/Dialect/Utils/StaticValueUtils.h#L72